src/app/app.component.ts

```diff
@@ -1,22 +1,36 @@
-import { Component } from "@angular/core";
-import { Platform } from "ionic-angular";
-import { Splashscreen, StatusBar } from "ionic-native";
+import { Component, ViewChild } from "@angular/core";
+import { Platform, Nav, App } from "ionic-angular";
+import { Splashscreen, StatusBar, Deeplinks } from "ionic-native";
 import { LoginPage } from "../pages/login/login";
-
-import {AuthService} from "../providers/auth.service";
+import { AuthService } from "../providers/auth.service";
 
 @Component({
   templateUrl: "app.html",
 })
 export class MyApp {
   public rootPage = LoginPage;
+  @ViewChild(Nav) nav:Nav;
+
 
-  constructor(platform: Platform, public auth: AuthService) {
+  constructor(platform: Platform, public auth: AuthService, public app: App,
+  ) {
```

Comment:
bring down the counstructor on multiple lines

constructor(
   platform: Platform, 
   public auth: AuthService,
  public app: App,
 ) { }